uber/config.py

```diff
@@ -13,15 +13,9 @@ def mixin(cls, klass):
     def include_plugin_config(self, plugin_config):
         """Plugins call this method to merge their own config into the global c object."""
 
-        for attr, val in plugin_config.items():
-            if not isinstance(val, dict):
-                setattr(self, attr.upper(), val)
-
-        if 'enums' in plugin_config:
-            self.make_enums(plugin_config['enums'])
-
-        if 'dates' in plugin_config:
-            self.make_dates(plugin_config['dates'])
```

Comment:
> This started because I wanted to define a new ribbon called dealer_asst_ribbon in the Anthrocon plugin... Technically we could skip defining enums in plugins for now and just make sure to add it to Anthrocon's development.ini

FWIW, that was the original intended approach for different events add their own config.  I wasn't really sure whether it was better or worse from a design/intuition perspective to put it there vs having each plugin add onto existing enums.  The main pros and cons that occurred to me were
- Having these options all one place might be less confusing.  "What ribbons options exist, and where does this particular ribbon option come from" can be answered by looking in a single plugin, namely the plugin where the concept of ribbons is defined.
- Letting each plugin define their own enums might be more intuitive, since the plugin where ``Dealer Assistant" is used is also the plugin where it's defined.

I was kind of on the fence about it, but if you think that the second option is definitely clearer/better/intuitive-er then we can definitely support that.  It's not a huge change to just modify the lists/dicts in place and ensure that the code which makes use of them knows that they can change.

In the meantime you can just set that ribbon in the core ``development.ini`` until we get all those kinks worked out, so this shouldn't hold up anyone's deployment.

> Thaeli mentioned earlier taking the c object definition out of runtime into a separate command, something like ``sep load_config_options``.

That seems like a huge refactor, much more work than just modifying existing options in-place in the ``c.make_enum()`` method.  I'm theoretically open to anything since the proof is in the pudding as far as clean/intuitive/working code, but I wouldn't want to do anything that drastic just to support enum merging.